doped/utils/parsing.py

```
@@ -1872,20 +1892,26 @@ def total_charge_from_vasprun(vasprun: Vasprun, charge_state: int | None) -> int
 
     try:
-        if vasprun.incar.get("NELECT") is None:
+        if get_nelect_from_vasprun(vasprun) is None:
             auto_charge = 0  # neutral if NELECT not specified
 
         else:
-            nelect = vasprun.parameters.get("NELECT")
-            neutral_nelect = get_neutral_nelect_from_vasprun(vasprun)
-
+            nelect = get_nelect_from_vasprun(vasprun)
+            if code == 'vasp':
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)